projects/wireshark/build.sh

```diff
@@ -83,27 +45,4 @@ sed -i '/AC_WIRESHARK_GCC_FORTIFY_SOURCE_CHECK/d' configure.ac
 
 make "-j$(nproc)"
 make install
-
-WIRESHARK_FUZZERS_COMMON_FLAGS="-lFuzzingEngine \
-    -L"$WIRESHARK_INSTALL_PATH/lib" -lwireshark -lwiretap -lwsutil \
-    -Wl,-Bstatic `pkg-config --libs glib-2.0` -pthread -lpcre -lgcrypt -lgpg-error -lz -Wl,-Bdynamic"
-
-for dissector in $FUZZ_DISSECTORS; do
-  generate_fuzzer "${dissector}" -DFUZZ_DISSECTOR_TARGET=\"$dissector\"
-done
-
-for dissector in $FUZZ_IP_PROTO_DISSECTORS; do
-  generate_fuzzer "ip_proto-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"ip.proto\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_TCP_PORT_DISSECTORS; do
-  generate_fuzzer "tcp_port-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"tcp.port\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_UDP_PORT_DISSECTORS; do
-  generate_fuzzer "udp_port-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"udp.port\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_MEDIA_TYPE_DISSECTORS; do
-  generate_fuzzer "media_type-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"media_type\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
+$SRC/wireshark/tools/oss-fuzzshark/build.sh all
```

Comment:
This is fine as is, but I would encourage you to make one more step 
and make the fuzz targets build as part of `make` so that they are always built and tested. 
You don't have to link them against libFuzzer or other fizzing engines -- instead you may link them against something like https://github.com/llvm-mirror/llvm/tree/master/lib/Fuzzer/standalone
Then you'll be able to use those build targets for reproducing oss-fuzz bug reports and for regression testing,
w/o having to call this .sh script. 

More: https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md